icu4c/source/common/unicode/localematcher.h

```diff
@@ -0,0 +1,590 @@
+// © 2019 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html#License
```

Comment:
I'll leave it up to the C++ API change report to make sure that this is consistent with the API proposal.

Reply:
I sent the update from copy-paste of the code, so fingers crossed :-)